services/invitations/src/simcore_service_invitations/services/invitations.py

```diff
@@ -64,8 +64,7 @@ def _build_link(
     r = URL("/registration").include_query_params(invitation=code_url_safe)
 
     # Adds query to fragment
-    base_url = f"{base_url}/"
-    url = URL(base_url).replace(fragment=f"{r}")
+    url = URL(f"{base_url.rstrip('/')}/").replace(fragment=f"{r}")
```

Comment:
`starlette.datastructures.URL` takes care of all `/`, so it is redundant

```suggestion
    url = URL(base_url).replace(fragment=f"{r}")
```

Reply:
Since it's an equivalent implementation and we need the PR in (to be hot fixed) will merge as is.